change/@fluentui-tokens-a097aef6-9fbd-4502-aa6d-52dd1fa44415.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33066&commitId=0f3794220638bebf72f76672d33831ac594b396c&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.Badge Mask RTL.chromium.png|1| Changed|
|Avatar Converged.badgeMask.chromium.png|5| Changed|

</details>


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay RTL.chromium.png|3279| Changed|

</details>


  <details><summary><b>Link Converged - Rendered as anchor</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as anchor.Inverted Disabled High Contrast.chromium.png|90329| Changed|
|Link Converged - Rendered as anchor.Inverted High Contrast.chromium.png|90514| Changed|

</details>


  <details><summary><b>Link Converged - Rendered as button</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as button.Inverted Disabled High Contrast.chromium.png|90329| Changed|
|Link Converged - Rendered as button.Inverted High Contrast.chromium.png|90514| Changed|

</details>


  <details><summary><b>Link Converged - Rendered as span</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as span.Inverted High Contrast.chromium.png|90514| Changed|
|Link Converged - Rendered as span.Inverted Disabled High Contrast.chromium.png|90329| Changed|

</details>


  <details><summary><b>PresenceBadge Converged - inverted background</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - inverted background.Default High Contrast.chromium.png|77386| Changed|

</details>


  <details><summary><b>TagPicker</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|TagPicker.Appearance High Contrast.chromium.png|622795| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>